src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -1183,6 +1183,8 @@ getAttributionJson apiOpts ProjectRevision{..} = fossaReq $ do
             =: True
           <> "dependencyInfoOptions[]"
             =: packageDownloadUrl
+          -- Large reports can take over a minute to generate, so increase the timeout to 10 minutes
+          <> responseTimeoutSeconds 600
```

Comment:
This seems fine, but I think it would be good to have a top-level `defaultResponseTimeout` and refer to that rather than write 600 in both places.

Reply:
The default is actually 30 seconds (from the library, and we are overriding it here.

The default comes from the Network.HTTP.Req library. Search for "responseTimeout" on this page: https://hackage.haskell.org/package/req-3.13.2/docs/Network-HTTP-Req.html#g:12